StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp11/DocumentationRules/SA1648CSharp11UnitTests.cs

```diff
@@ -3,9 +3,40 @@
 
 namespace StyleCop.Analyzers.Test.CSharp11.DocumentationRules
 {
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis.Testing;
     using StyleCop.Analyzers.Test.CSharp10.DocumentationRules;
+    using Xunit;
 
     public partial class SA1648CSharp11UnitTests : SA1648CSharp10UnitTests
     {
+        [WorkItem(3595, "https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3595")]
+        [Theory]
+        [InlineData("abstract void TestMethod();", "void TestMethod() {}")]
+        [InlineData("virtual void TestMethod() {}", "void TestMethod() {}")]
+        [InlineData("abstract int TestProperty { get; set; }", "int TestProperty { get; set; }")]
+        [InlineData("virtual int TestProperty { get; set; }", "int TestProperty { get; set; }")]
+        [InlineData("abstract event System.Action TestEvent;", "event System.Action TestEvent;")]
+        [InlineData("virtual event System.Action TestEvent;", "event System.Action TestEvent;")]
+        public async Task TestCorrectMemberInheritDocFromStaticAbstractOrVirtualMemberInInterfaceAsync(string interfaceMember, string classMember)
```

Comment:
❓  Can we also test explicit interface implementations?

Reply:
Updated